### ISSUE TITLE
could not update translation without term

### ISSUE THREAD

**Issue by srl295** [mozilla/pontoon#3076]
1. setup Pontoon 2a0287244042fdecb65a40e0bcca766fa0cd9614 using [dev setup docs here](https://mozilla-pontoon.readthedocs.io/en/latest/dev/setup.html)
2. tried to add a string using the http://localhost:8000/admin/projects/terminology/strings/ page
3. then in translating, clicking on the Save button such as http://localhost:8000/csw/terminology/common/?string=71 'spins' briefly but doesn't mark as saved
4. browser console shows a 500 internal error, and docker console shows
 
```shell
pontoon.base.models.Entity.term.RelatedObjectDoesNotExist: Entity has no term.
[ERROR:django.server] 2024-01-20 06:54:05,420 "POST /translations/create/ HTTP/1.1" 500 18782
```

(above error seen mentioned in another issue but that's all)

5. however, trying to leave the area said 'you have unsaved changes!'
6. i went to http://localhost:8000/a/terminology/term/add/ and somehow added my term. Then i was able to save.

-----

so it seems that:
- the 500 internal err should be propagated to the UI somehow
- otherwise there may be user error here

thanks!

**Comment by mathjazz** [comment]
Hey @srl295, please see https://mozilla-l10n.github.io/documentation/tools/pontoon/adding_new_terminology.html on how to add new terms.

**Comment by srl295** [comment]
@mathjazz thanks, though:

- should this be documented within pontoon?
- are the UI issues an internal error within pontoon?

**Comment by mathjazz** [comment]
> * should this be documented within pontoon?

Yes. That's covered by https://github.com/mozilla/pontoon/issues/2214.

> * are the UI issues an internal error within pontoon?

Good point. This is still a bug that should be fixed.

**Comment by mathjazz** [comment]
The error was fixed in #3097.
